### ISSUE TITLE
[Bug] Features button missing, volume control with scroll not working, automatic theatre not working and more

### ISSUE THREAD

**Issue by rado84-github** [VampireChicken12/youtube-enhancer#349]
**Describe the bug**
Features button missing, volume control with scroll not working and more

**To Reproduce**
I think it's better to see it for yourself, than explaining it step by step:
[simplescreenrecorder-2024-02-15_21.36.03.mkv.zip](https://github.com/VampireChicken12/youtube-enhancer/files/14301485/simplescreenrecorder-2024-02-15_21.36.03.mkv.zip)
This is the search results URL in case you need to test it yourself:
https://www.youtube.com/results?search_query=%D0%B1%D1%80%D0%B0%D1%82%D1%8F+%D0%BC%D0%B0%D0%BD%D0%B3%D0%B0%D1%81%D0%B0%D1%80%D1%8F%D0%BD+cdroma

**Expected behavior**
When a video is loaded, the last used sound volume levels should be loaded, automatic theatre mode should expand the video throughout the entire length of the screen (like this: https://i.imgur.com/HclbrNk.png).

**What actually happens**
This is what actually happens when a video is loaded:
![Picture_created_15-02-2024_21-34-56](https://github.com/VampireChicken12/youtube-enhancer/assets/41172201/807b5d0b-832f-407a-9fc8-f0751dc71b37)
Also, "Remember last used sound volume" doesn't work, despite that it has been enabled in the settings, as you see in the above screenshot.

**Desktop (please complete the following information):**

- OS: Linux
- Browser: Firefox 122.0.1
- Version: 1.23.2

**Additional context**
Basically the only thing from the whole extension that seems to be working is the custom CSS theme.

**Comment by mist8kengas** [comment]
I found out what was causing the bug, now just need to find out how to reproduce this 100% of the time to be able to find a fix for it.
